hrabsence/CRM/HRAbsence/BAO/HRAbsenceType.php

```diff
@@ -117,6 +77,67 @@ public static function create($params) {
   }
 
   /**
+   * Creates Activity Types based on $params and $activityTypes
+   *
+   * @param  array $activityTypes
+   * @param  array   $params
+   * @param  boolean $isDebit  ('true' for 'debit' and 'false' for 'credit')
```

Comment:
The same thing applies here. You either remove the description or the parenthesis